civis/ml/_model.py

```diff
@@ -134,7 +134,8 @@ def _retrieve_file(fname, job_id, run_id, local_dir, client=None):
     """Download a Civis file using a reference on a previous run"""
     file_id = cio.file_id_from_run_output(fname, job_id, run_id, client=client)
     fpath = os.path.join(local_dir, fname)
-    os.makedirs(os.path.dirname(fpath), exist_ok=True)
+    if not os.path.exists(os.path.dirname(local_dir)):
```

Comment:
Does fname contain a path? If not, I don't need the dirname because I already have the path?